gatsby-site/src/components/Subscribe/subscribe.css

```diff
@@ -0,0 +1,85 @@
+:root {
+    --colarkfontcolor: #0111FC;
+    --colarkfont: 'Mukta Malar', sans-serif;
+    --background: #FFFFFF;
+}
+
+.subscription-container {
+    text-align: right;
+    width: 100%;
+    font-family: var(--colarkfont);
+}
+
+.form-control {
+    display: inline-block;
+    margin: 0 auto;
+    border: 5px solid white;
+    color: rgba(0, 0, 0, 0.25);
+    width: 400px;
+    border-radius: 0px;
+}
+
+.subscription-header {
+    text-align: left;
+    font-size: 24px;
+    color:white;
+    margin-left: 250px;
+}
+
+.subscription-input {
+    width: 500px;
+    text-align: center;
+    border: 3px solid white;
+    margin: 0 auto;
+    font-weight: 200px;
+    box-shadow: 5px 10px 18px rgba(0, 0, 0, .5);
+    background: white;
+    border-radius: 5px;
+}
+
+.subscribesubmit {
+    width: 94px;
+    border-radius: 5px;
+}
+
+.body-container-minus-footer {
```

Comment:
Is this style definition used elsewhere? It doesn't look like it is used in this file.